core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -115,6 +116,11 @@
     public static ComparisonExpression ltNum(final ValueExpression p) { return new LtNum(null, p); }
     public static ComparisonExpression ltNum(final ValueExpression c, final ValueExpression p) { return new LtNum(c, p); }
 
+    public final static Reducer ADD_REDUCER = new Reducer() { @Override public ValueExpression reduce(final ValueExpression l, final ValueExpression r) { return add(l, r); } };
```

Comment:
Are these worth promoting to the `Shorthand`? I was more thinking about making these available for testing. Although these might disappear again after we implement #50.